README.md

```diff
@@ -4,7 +4,20 @@
 
 This project is an open source tool to convert TIMES model Excel input files to DD format ready for processing by GAMS.  The intention is to make it easier for people to reproduce research results on TIMES models.
 
-TIMES is an energy systems model generator from the International Energy Agency that is used around the world to inform energy policy - https://iea-etsap.org/index.php/etsap-tools/model-generators/times
+The project began as a [Hackathon](https://news.microsoft.com/life/hackathon/) project hence the repo being in the Microsoft organisation, however Microsoft are not funding this work. Further work has been done as part of company Hackathons and the rest in people's own time (both Microsoft employees and not).
```

Comment:
I don't think you should include this sentence.  What message are you trying to convey?

Reply:
That the project isn't funded by Microsoft. People could be forgiven for thinking the opposite given it's location

Reply:
Why does anyone care about that?  Is it something to do with the level of support?

Reply:
That's the most important reason but also I've put a lot of my own time into this that I'm not happy for Microsoft to get the credit for

Reply:
The "own time" sentence covers that.

Reply:
Is this better? Update doesn't show for me at the top of this conversation for some reason